Week1/MAKEME.md

```diff
@@ -35,33 +35,106 @@ Write commands to do following:
+> For all the following exercises create a new .js file. Try to find a proper name for each file or make a small comment about what it does inside for future reference. 
 
 1. Declare a variable `x` and initialize it with an integer.
-
-2. How do you round the number 7.25, to the nearest integer?
-
-3. Create a array called `colors` with the strings red, green and blue inside.
-
-4. How can you find the length of the string you just created?
-
-5. Write a program that checks the types of two variables and prints out `SAME TYPE` if they are the same type. 
-For example: 
+  1.1 First, _declare_ your variable `x`.
+  1.2 Add a console.log statement that explains what _you think_the value of `x` is, like in this example:
+  ```js
+  // TODO -> here you initialize your variable
+  console.log('the value of my variable x will be: whateverYouThinkItWillLog')
+  ```
+  1.3 Add a console.log statement that logs the value of `x`
+  1.4 Now _initialize_ your variable `x` with an integer
+  1.5 Now add a console.log statement that explains what _you think_ the value of `x` is.
+  1.6 Add a console.log statement that logs the value of `x`.
+  Steps to be taken:
+  
+  ```js
+  // TODO -> here you declare your variable
+  console.log('the value of x will be: whateverYouThinkItWillLog')
+  // TODO -> log the actual value of x
+  // TODO -> here you initialize your variable
+  console.log('the value of x will be: whateverYouThinkItWillLog')
+  // TODO -> log value of x again
+  ```
+
+2. Declare a variable `y` and assign a string to it.
+  2.1 Write a console.log statement in which explain what you think the value of the string is
+  2.2 Now console.log the variable `y`.
+  2.3 Now assign a new string to the variable `y`
+  2.4 Just like you did before write a console.log statement that explains what you think will be logged to the console.
+  2.5 Now console.log `y` again.
+  ```js
+  // TODO -> here you declare AND assign your string
+  console.log('the value of my string will be: whateverYouThinkItWillLog')
+  // TODO -> log the actual value of the string to the console
+  // TODO -> assign a new value to your variable x
+  console.log('the value of my string will be: whateverYouThinkItWillLog')
+  // TODO -> log the actual value of the string to the console
+  ```
+
+3. How do you round the number 7.25, to the nearest integer?
+  3.1 Declare a variable `z` and assign the number 7.25 to it.
+  3.2 Console.log `z`.
+  3.3 Now round `z` to the nearest integer.
+  3.4 Console.log `z` again.
+
+4. Arrays!
+  4.1 Declare an empty array (you can decide on how to call it yourself).
+  4.2 Write a console.log statement in which explain what you think the value of the array is.
+  4.3 Console.log your array.
+  4.4 Create an array that has your favorite animals inside
+  4.5 Log your array
+  4.6 Ads Daan's favorite animal (baby pig) to the existing array
```

Comment:
Typo: Ads -> Add

Reply:
I assume you expect students to use `push` rather than modifying their array initialization in the source code. In that case we could say: Add a statement to adds Daan's...